src/drtsans/mono/biosans/api.py

```diff
@@ -1205,6 +1206,9 @@ def reduce_single_configuration(
     absolute_scale = reduction_config["StandardAbsoluteScale"]
     time_slice = reduction_config["useTimeSlice"]
     time_slice_transmission = reduction_config["useTimeSlice"] and reduction_config["useTimeSliceTransmission"]
+    sample_trans_error_tol = None
+    if time_slice_transmission:
+        sample_trans_error_tol = reduction_input["sample"]["transmission"]["errorTolerance"]
```

Comment:
please include a comment here that we're checking for transmission tolerance errors only when slicing because this is the only observed use-case of poor statistics for transmission calculations